src/main/scala/org/apache/spark/streamdm/streamDMJob.scala

```diff
@@ -33,13 +34,24 @@ object streamDMJob {
 
     //configuration and initialization of model
     val conf = new SparkConf().setAppName("streamDM")
-    conf.setMaster("local[2]")
 
-    val ssc = new StreamingContext(conf, Seconds(10))
+    var newArgs = args.clone()
```

Comment:
newArgs is not a good name.

Reply:
Hi Zhang, I changed the variable to paramsArgs (arguments for parameters). I think it makes more sense.